e2etest/layer2_test.go

```diff
@@ -307,3 +433,26 @@ func getEndpointHostName(ep string, exec executor.Executor) (string, error) {
 
 	return res, nil
 }
+
+// taken from: poolCount returns the number of addresses in the pool.
+func ipRangeCount(ipRanges []string) (int64, error) {
```

Comment:
I'm not sure since the one from internal expects to get an AddressPool from internal: https://github.com/metallb/metallb/blob/f79e0a2e8c0dbe7c26c46d479e802bdd187ec262/internal/allocator/allocator.go#L323
but now that I look at it again I don't remember why I rewrote it to accept []string instead of a pool :man_shrugging:

Reply:
changed it a bit to look more like the internal one but still not sure we can just export the original